addons/packages/fluxcd-kustomize-controller/0.21.1/README.md

```diff
@@ -0,0 +1,208 @@
+      - Retrieving installation details for fluxcd-kustomize-controller...
+        NAME:                    fluxcd-kustomize-controller
+        PACKAGE-NAME:            kustomize-controller.fluxcd.community.tanzu.vmware.com
+        PACKAGE-VERSION:         0.21.1
+        STATUS:                  Reconcile succeeded
+        CONDITIONS:              [{ReconcileSucceeded True  }]
+       ```
+
+       Verify that `STATUS` is `Reconcile succeeded`
+
+       ```shell
+       kubectl get pods -n kustomize-system
+       ```
+
+       For example:
+
+       ```shell
+       kubectl get pods -n kustomize-system
+       NAME                                    READY   STATUS    RESTARTS   AGE
+       kustomize-controller-7d96857bfc-hcbmk   1/1     Running   0          3m36s
+       ```
+
+       Verify that `STATUS` is `Running`
+
+## Try kustomize-controller
+
+1. Verify all the objects are installed:
+
+    This package would create a new namespace where all the elements of fluxcd will be hosted called `kustomize-system`
+
+    you can verify the main components of `kustomize-controller` were installed by running:
+
+    ```shell
+    kubectl get all -n kustomize-system  
+    NAME                                        READY   STATUS    RESTARTS   AGE
+    pod/kustomize-controller-7d96857bfc-hcbmk   1/1     Running   0          6m12s
+
+    NAME                                   READY   UP-TO-DATE   AVAILABLE   AGE
+    deployment.apps/kustomize-controller   1/1     1            1           6m12s
+
+    NAME                                              DESIRED   CURRENT   READY   AGE
+    replicaset.apps/kustomize-controller-7d96857bfc   1         1         1       6m12s
+    ```
+
+    you should get something really similar!
+
+2. Verify all the CRD were installed correctly:
+
+    The way you would communicate with `kustomize-controller` would be through its CRDs, this will be your main action point.
+
+    In order to check all the CRDs were installed you can run:
+
+    ```shell
+    kustomize.kustomize.toolkit.fluxcd.io                                     2022-04-10T16:12:13Z
+    ```
+
+3. Try one simple example:
+
+    Try one quick example yourself, so you can check everything is working as expected
+
+    - Let's consume a `GitRepository` and `Kustomization` object,
+
+      - Create the following `sample.yaml` file:
+
+       ```yaml
+       apiVersion: source.toolkit.fluxcd.io/v1beta1
+       kind: GitRepository
+       metadata:
+       name: podinfo
+       namespace: flux-system
+       spec:
+       interval: 1m
+       url: https://github.com/stefanprodan/podinfo
+       ---
+       apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
+       kind: Kustomization
+       metadata:
+       name: podinfo
+       namespace: flux-system
+       spec:
+       interval: 5m0s
+       path: ./kustomize
+       prune: true
+       sourceRef:
+       kind: GitRepository
+       name: podinfo
+       targetNamespace: flux-system
+       ```
```

Comment:
This YAML is not formatted/indented correctly and gives a validation error. Can you please correct it?